tests/cli/fsm/autosuggest_test.py

```diff
@@ -61,7 +61,7 @@ def test_suggest_smartstack_proxy_port(self):
             55555,  # bogus out-of-range value
         ]
 
-        def get_smarstack_proxy_port_from_file_side_effect(*args):
+        def get_smarstack_proxy_port_from_file_side_effect():
```

Comment:
This is important -- this function is standing in for a function that takes arguments, and you've changed the signature. I think this is what's causing the unit test failures.

However, the way we're doing this seems kinda wonky anyway. I don't think we need this function at all - I believe we could just pass `side_effect=get_smartstack_proxy_port_from_file_returns` on line 66.